.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,66 +24,62 @@
 
-* https://deploy-preview-{your pr number}--boosted.netlify.app/
+* https://deploy-preview-{your_pr_number}--boosted.netlify.app/
 
 ### Checklist
 
 <!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 
+#### Contribution
+
 - [ ] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
+
+#### Design
+
+- [ ] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
+- [ ] My change is compatible with responsive display
+
+#### Development
+
 - [ ] My code follows the code style of the project _(using `npm run lint`)_
-- [ ] My change introduces changes to the documentation
-- [ ] My change introduces changes to the migration guide
-- [ ] I have updated the documentation accordingly
-- [ ] I have added tests to cover my changes
+- [ ] I have added JavaScript unit tests to cover my changes
 - [ ] All new and existing tests passed
 
-<!------------------------->
-<!-- /!\ Core Team Only -->
-<!------------------------->
+#### Documentation
 
-<!-- Uncomment the following for a feature DoD -->
+- [ ] My change introduces changes to the documentation or I have updated the documentation accordingly
 
-<!--
-### Development
-
-- [ ] Should match specs (eg. either the Web UI Kit or any pattern from the WAI — or both…)
-- [ ] Docs added:
-  - including the "Sass" part using `scss-docs` shortcode
-  - in /about/overview/#custom-components if it is a new Orange custom component
-  - in /getting-started/introduction/#components if it is a new Orange custom component that requires JavaScript (and Popper)
-  - in /customize/overview#csps-and-embedded-svgs if it is a new Orange custom component that includes embedded SVGs in our CSS
-  - in /forms/validation/?#supported-elements if it is a new Orange custom component that is a form control
-  - in /forms/overview/ if it is a new Orange custom component that is a form control
-- [ ] (Optional) If this feature is a new component, it probably must appear in Storybook
-- [ ] Check (and fix) RTL version
-- [ ] Run linters
-- [ ] Run compilers
-- [ ] Tests added for JS-side
-- [ ] Run tests
-- [ ] Manually run BrowserStack test
-- [ ] Manually run Percy test
-- [ ] Cross-browser test:
-  - [ ] Chrome
-  - [ ] Firefox (including ESR)
-  - [ ] Edge
-  - [ ] Safari
-  - [ ] iOS Safari
-  - [ ] Chrome & Firefox on Android
-- [ ] Responsive tests: desktop and small screens
-- [ ] Clean up the branch using `rebase -i`
-- [ ] Commited with `feat(…): …` message
-- [ ] Mention it in Migration Guide (if `back-from-v4`): renamed variables, changes in markup requirement,  etc.
-
-### Reviews
+<details>
+  <summary>Detailed info</summary>
+  <ul>
+    <li>including the "Sass" part using <code>scss-docs</code> shortcode</li>
```

Comment:
At the end IMO we can't explain all the possibilities. I suppose this ended up here because it was always forgotten. Maybe a part of what's contained in the `<details>` should end up in the Wiki with a more detailed version.

Reply:
Removed it to put it in the developer guide (wiki page)